src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPAddressCollectionImpl.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
```

Comment:
I believe this is unused. There's a shared one that Windows also uses that I think I switched to.